src/odemis/odemisd/test/main_test.py

```diff
@@ -309,6 +309,7 @@ def test_persistent_data(self):
         stage = model.getComponent(role='stage')
         stage.speed.value = speed
 
+        time.sleep(1)  # make sure the backend is done writing the VA values to the settings file
```

Comment:
Also need to add to similar places. Cf line 400

Reply:
added it to line 400